apps/src/turtle/api.js

```diff
@@ -203,6 +203,10 @@ ArtistAPI.prototype.turnLeft = function (angle, id) {
   this.log.push(['RT', -angle, id]);
 };
 
+ArtistAPI.prototype.pointTo = function (angle, id) {
+  this.log.push(['PT, angle, id']);
```

Comment:
This pushes an array with a single item?  I think the quotes might be mismatched from where you want them.